nipy/algorithms/statistics/formula/tests/test_formula.py

```diff
@@ -43,24 +43,24 @@ def test_terms():
 def test_getparams_terms():
     t = F.Term('t')
     x, y, z = [sympy.Symbol(l) for l in 'xyz']
-    yield assert_equal, set(F.getparams(x*y*t)), set([x,y])
-    yield assert_equal, set(F.getterms(x*y*t)), set([t])
+    assert_equal(set(F.getparams(x*y*t)), set([x,y]))
+    assert_equal(set(F.getterms(x*y*t)), set([t]))
```

Comment:
I've been gradually removing test generators from the code because the errors are harder to debug.  Here I did it as part of refactoring to get ready to add the new tests, no special reason.